commander3/src/comm_n_rms_mod.f90

```diff
@@ -79,6 +79,10 @@ function constructor(cpar, info, id, id_abs, id_smooth, mask, handle, regnoise,
     ! General parameters
     allocate(constructor)
 
+    ! NEED TO ADD A TEST
+    ! Check that when you're doing matrix multiplication, the number of maps is
+    ! consistent with the input and the noise covariance matrix.
+
```

Comment:
Has this test been implemented?

Reply:
It looks like it has been for the `matul` subroutines

Reply:
Has been implemented